### PR TITLE
BZ-1740977: Added example scenario for cluster limits description.

### PR DESCRIPTION
**Pull request by sagidlow**
Applies to 4.5+
BZ Link: https://bugzilla.redhat.com/show_bug.cgi?id=1740977
QE Approval Needed: @skordas 
Preview Link: https://deploy-preview-31285--osdocs.netlify.app/openshift-enterprise/latest/scalability_and_performance/planning-your-environment-according-to-object-maximums.html#cluster-maximums-major-releases_object-limits

### CODE REVIEW

File: modules/openshift-cluster-maximums-major-releases.adoc [hunk 1]
```diff
@@ -8,6 +8,11 @@
 Tested Cloud Platforms for {product-title} 3.x: {rh-openstack-first}, Amazon Web Services and Microsoft Azure.
 Tested Cloud Platforms for {product-title} 4.x: Amazon Web Services, Microsoft Azure and Google Cloud Platform.
 
+[NOTE]
+====
+Some limits were tested independently, therefore a comparison between rows should not be made.
```
Comment (bergerhoffer) [inline modules/openshift-cluster-maximums-major-releases.adoc:13]: Consider active - "therefore do not compare between rows" or "therefore do not make a comparison between rows"

File: modules/openshift-cluster-maximums-major-releases.adoc [hunk 2]
```diff
@@ -65,3 +70,20 @@ Tested Cloud Platforms for {product-title} 4.x: Amazon Web Services, Microsoft A
...
+6000 namespaces x 2 pods = 12000 pods
+----
+
+In total, you have 27,000 pods.
+
+----
+27000 pods / 250 pods per node = 108 nodes
+----
+
+The limit is 2,000 per node, therefore a 200 node cluster would work. However, there are 11,000 namespaces when the limit is 10,000. etcd might have issues handling this many active projects.
```
Comment (skordas) [inline modules/openshift-cluster-maximums-major-releases.adoc:89]: 2000 here is a number of nodes per cluster (it looks I wrote this wrong in BZ).
maybe something like:
Node maximum for cluster is 2000, therefore a 200 node cluster would work. However, there are 11,000 namespaces when the maximum is 10,000....

Comment (sagidlow) [reply modules/openshift-cluster-maximums-major-releases.adoc]: I updated per your comment, let me know if that looks ok?

**Review by skordas — APPROVED** [review]

**Comment by sagidlow** [comment]
After speaking with Eric Rich, this PR is being closed per his advice as well as the PM's. The Bug will subsequently also be closed as 'Won't Fix'.

**Pull request closed without merging**